docs/en/stack/ml/df-analytics/flightdata-regression.asciidoc

```diff
@@ -0,0 +1,357 @@
+[role="xpack"]
+[testenv="platinum"]
+[[flightdata-regression]]
+=== Predicting flight delays with {reganalysis}
+
+Let's try to predict flight delays by using the 
+{kibana-ref}/add-sample-data.html[sample flight data]. We want to be able to use 
+information such as weather and location of the destination and origin, flight 
+distance and carrier to predict the number of minutes delayed for each flight. 
+As it is a continuous numeric variable, we'll use {reganalysis} to make the 
+prediction.
+
+This dataset consists of sample data that has been manually created and it 
+contains inconsistencies. For example, a flight can be both delayed and 
+canceled. Nonetheless, the use case is relevant and it is a good example of how 
+the quality of input data will affect the quality of results.
```

Comment:
The tone of the para does not sit quite right, possibly a bit arrogant/unfriendly. Can we soften this please. Perhaps ...

```
We have chosen this dataset as an example because it is easily accessible
for Kibana users and the use case is relevant. However the data has been manually 
created and contains some inconsistencies. For example, a flight can be both delayed 
and canceled. Please remember that the quality of your input data will affect the quality
of results.
```